app/client/service_runner.py

```diff
@@ -34,9 +34,7 @@ def run_master(self):
         self._logger.info('Running master on {}'.format(self._master_url))
         if self.is_master_up():
             return
-        cmd = '{} master --port {} &'.format(
-            self._main_executable,
-            self._port(self._master_url))
+        cmd = [self._main_executable, 'master', '--port', self._port(self._master_url)]
```

Comment:
Actually I am going to remove '&' here since it doesn't seem matter here and '&' is not cross-platform.